### DISCUSSION TITLE
Limitless by Jim Kwik

### DISCUSSION THREAD

**Discussion by trainfarb** [hu3agency/ep0_daoTeaching#53] [#b0mission]
# Welcome!
This is the discussion thread for sharing book quotes from The Polymath by Waqās Ahmed. These book quotes are meant to set the tone for how the LP0_polymathProcess DAO wants Humanity's Language Institution to evolve. In the huniverse, these quotes define the water & soil quality of LP0's tree, how poly (the brandedBeing) speaks, and the advice being sent from our community into the parallel reality through our decentralized representative, hu.

- [Spotify Playlist](https://open.spotify.com/playlist/25WElf3Mx7mSjNDWYOkmsv?si=3251d7e038f1419b)

Feel free to share book quotes below. Designers from our community will transform the quotes you choose into beautiful posts that anyone from the community can share on social media to help our collective voice spread. These posts will be sold as NFTs to compensate the people involved and raise money for LP0's future music festivals.

When sharing any of the designed posts below, please use #b0media.

**Comment by trainfarb** [comment]
> “The first element of the three-part Limitless Model is Mindset, which is the mental attitude or disposition that predetermined a person’s responses to and interpretations of situations. Mindset is made up of beliefs, assumptions, and attitudes we hold about ourselves and the world around us. All behavior is driven by belief, so before we address how to learn, we must first address the underlying beliefs we hold about what is possible.” p66

**How can humanity use this?** We can start exploring what the predetermined responses are for our institutions. What are the beliefs that are engrained in our institutions, and subsequently, in each of us. We’re all brainwashed by our CULTure (for better or worse). We can use this quote to become more aware of this.

**Why must we use this?** When we allow our institutions to continue operating on autopilot, we’ll all be trapped in the mechanisms that run the world regardless of how we feel about them deep down. We need to train our institutions to explore this so we’re all taught to explore it from a young age.

**When will we use this?** Whenever it seems everyone is blindly aligned on an issue, we will explore how it might be related to underlying programming. Then, we’ll make a call about how we want to learn from the situation and respond accordingly.

**Comment by trainfarb** [comment]
> Part of being limitless is learning to let go of the fear of criticism from other people. p 101

**How can Humanity use this?** People need to learn how to make mistakes in speech and action so we can learn from our mistakes and grow collectively.

**Comment by trainfarb** [comment]
![image](https://user-images.githubusercontent.com/8133349/156934707-6f50c22a-c9c3-4210-ba63-ecfc111d5a28.jpeg)

What is humanity's purpose?

**Comment by trainfarb** [comment]
> The longer you go without sleep, the harder it becomes to maintain a sense of reality—or motivation, for that matter. Lack of sleep compromised all of your cognitive skills, your focus, your memory, and your overall brain health. A common contributing factor for depression and many mood disorders is lack of sleep.

What is society's equivalent to lack of sleep?

**Comment by trainfarb** [comment]
> Part of being limitless is learning to let go of the fear of criticisms from other people.

P101 from Limitless by **Jim Kwik**

_How does humanity criticize itself?_

**Comment by trainfarb** [comment]
![image](https://user-images.githubusercontent.com/8133349/157922336-1a60b102-671f-482c-96cc-97fbf33e4541.jpeg)

**Comment by trainfarb** [comment]
> Our negative habits operate with the same level of automaticity as our healthy ones.

p156: **Limitless** by Jim Kwik

_What's humanity's biggest negative habit? What's yours?_

**Comment by trainfarb** [comment]
> Too many of us don't come close to our capacities because we are too afraid of making a mistake. Instead of looking at mistakes as proof of failure, take them as proof that you are trying.

p96 from _Limitless_ by **Jim Kwik**

_What is some proof that humanity is trying? (there's plenty 🤪)_

**Comment by trainfarb** [comment]
> "Concentrstion is at the crux of all human success and endeavor," Hindu priest, entrepreneur, and former monk Dandapani told me during one of my podcasts. "If you can't concentrate, you can't manifest." What Dandapani is saying is that concentration is a critical component of anything you want to accomplish.

p184: Limitless by **Jim Kwik**

_Does humanity know how to collectively concentrate on meaningful goals? Should we practice?_

**Comment by trainfarb** [comment]
> Numerous studies link music to learning. The arousal-and-mood hypothesis, introduced by Dr. E. Glenn Schellenberg, identifies a connection between music and mood and the subsequent connection between mood and learning, suggesting that music can put us in conditions that improve our ability to learn.

p204: Limitless by **Jim Kwik**

_How would humanity change if we embedded music into our institutions?_

**Comment by trainfarb** [comment]
> When we take responsibility for something, we are imbued with great power to make things better.

p67: Limitless by **Jim Kwik**

_What's "wrong" with humanity? What are we responsible for?_

**Comment by kdnolan** [comment]
Great quote!

**Reply by trainfarb** [reply]
This book is great!

**Comment by trainfarb** [comment]
> Any plan to make your learning limitless needs to include reading. Just as memory is foundational to nearly all brain function, reading is foundational to nearly all learning. If anyone tells you that the don't read, they're essentially saying, "I've stopped trying to learn."

p239: Limitless by **Jim Kwik**

_What books would empower us to transform our institutions?_

**Comment by trainfarb** [comment]
> A great imagination helps you see more possibilities in your life, and reading keeps your imagination on high alert.

p241: Limitless by **Jim Kwik**

_How is humanity's collective imagination?_

**Comment by trainfarb** [comment]
> Solving the underlying problem allows for a much more workable solution to the surface problem.

p273: Limitless by **Jim Kwik**

_Think of a recent challenge humanity faced. Did we focus on solving the underlying problem or the surface problem?_

**Reply by Patrick-T-Anderson** [reply]
Solving the surface problem is usually better for the confused goal of perpetuating profit.

Business tends to avoid solving the underlying problem completely, because that would put them out of business!

But solutions are not a threat when we own businesses *because we want the results* ([For Product](https://docs.google.com/document/d/1cRYLXIQnNWfXNU88UvxdlKgdLfOyoXSvl4xyp6xytMU/)), since, in that case, we no longer need to be concerned about profit because it is simply not the goal of those investors.

**Reply by trainfarb** [reply]
Ooo I like "for product" a lot! Thanks for sharing that link
